resources/prometheus/federation-config.yaml

```diff
@@ -30,6 +30,9 @@
       - apiserver_request:burnrate6h{job!~"central|scanner"}
       - apiserver_request_terminations_total{job!~"central|scanner"}
       - apiserver_request_total{job!~"central|scanner"}
+      - argocd_app_info{job!~"central|scanner"}
```

Comment:
If I'm not mistaken, this file is generated. Check `README.md` and `generate-federate-match.sh` for details.
Make sure your change will not be overridden after the next generation.